pandas/tests/plotting/test_warns.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+import pandas as pd
+import pandas.util.testing as tm
+
+from pandas.plotting import _converter
+from pandas.tseries import converter
+
+
+plt = pytest.importorskip('matplotlib.pyplot')
+
+
+def test_warns():
```

Comment:
Is this duplicate with the other test file?

Reply:
Yes, forgot to remove it from my first commit.